lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './command_line_in'
+require_relative './prompt'
+require_relative './board_presenter'
+
+class TicTacToe
+    def assign_token(taken_token)
+        taken_token == 'X' ? "O" : "X"
```

Comment:
Another variable that does nothing

Reply:
Yes, the token should be abstracted and stored elsewhere. I'll delete this line.